src/controllers/user.js

```diff
@@ -87,27 +87,26 @@ export default class UserController {
     }
   }
 
-  static async forgetPassword(req, res) {
+  static async forgetPassword(req, res, next) {
     const { email } = req.body;
     const foundUser = await _user.findOne({ where: { email } });
     if (foundUser) {
       const token = jwt.sign(
         { email: foundUser.email, _id: foundUser._id },
         foundUser.password,
         {
-          expiresIn: '24h',
+          expiresIn: "24h",
         }
       );
       const data = {
-        from: 'alexis2020@gmail.com',
+        from: "alexis2020@gmail.com",
```

Comment:
Why are you using your e-mail?

Reply:
this is not my email you can use whatever as long as it has email format!

Reply:
or if you want you can keep in environment variable and you call it out there!